javascript/stimulus_reflex.js

```diff
@@ -266,18 +273,28 @@ const getReflexRoots = element => {
 // - options
 //   * controller - [optional] the default StimulusReflexController
 //   * consumer - [optional] the ActionCable consumer
+//   * logging - [optional] option wheter reflex logging should be enabled or not
 //
 const initialize = (application, options = {}) => {
   const { controller, consumer } = options
   actionCableConsumer = consumer
   stimulusApplication = application
+  stimulusApplication.options = { ...defaultOptions, ...options }
```

Comment:
I wouldn't hang our options off of the Stimulus application given that this is only for StimulusReflex. I'd probably track this as a file scope level variable similar to what we're doing with [promises](https://github.com/hopsoft/stimulus_reflex/blob/master/javascript/stimulus_reflex.js#L24).

```js
const options = {
  logging: false
}

// ...

if (options.logging) {
  // ...
}

// ...

export default {
   initialize,
   register,
   options
 }
```